gp-templates/translations.php

```diff
@@ -32,7 +32,7 @@
 		<option value="" selected="selected"><?php _e( 'Bulk Actions', 'glotpress' ); ?></option>
 		<option value="approve"><?php _e( 'Approve', 'glotpress' ); ?></option>
 		<option value="reject"><?php _e( 'Reject', 'glotpress' ); ?></option>
-		<option value="fuzzy"><?php _e( 'Fuzzy', 'glotpress' ); ?></option>
+		<option value="fuzzy"><?php _e( 'Mark as fuzzy', 'glotpress' ); ?></option>
```

Comment:
Same as above.

Reply:
In this case, must be added a context, as the action of mark as Fuzzy is translated have a different translation from the Fuzzy status.
Maybe all these strings could have contexts for "Action", "Single Status", "Footer Label"